### PR TITLE
Drop support of protocol v1 (sw3)

### LINKED ISSUE #4176: Remove all 5.x protocol and sw3 header from documents and codes
According to 

### Deprecated services
Deprecated service(s) are the gRPC services SkyWalking used before. In SkyWalking v6, in order to match the common concepts in CloudNative world, these services are deprecated. Although there services are still supported at this moment, but it will keep in supported at least before the end of 2019.

___
Now, it is 2020, we should remove all these permanently. If no objection, I will try to do this.

### PR DESCRIPTION
**Pull request by kezhenxu94**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance
- [x] Chore

- Related issues

closes #4176 

protocol removal will be done in the protocol repository

### CODE REVIEW

**Comment by wu-sheng** [comment]
> data-collect-protocol also dropped or updated to latest?

I think should be before :)

**Comment by JaredTan95** [comment]
> protocol removal will be done in the protocol repository

Sorry, I missed this.

**Comment by wu-sheng** [comment]
@kezhenxu94 MySQL e2e test seems failure, maybe due to some 3rd party issue?

**Comment by kezhenxu94** [comment]
> @kezhenxu94 MySQL e2e test seems failure, maybe due to some 3rd party issue?

E2E MySQL test seems to fail in all PRs, I'm checking

**Comment by wu-sheng** [comment]
> E2E MySQL test seems to fail in all PRs, I'm checking

Yes. Thanks.

**Comment by wu-sheng** [comment]
We should clean https://github.com/apache/skywalking-data-collect-protocol of v1. But should we do after #4220?

**Comment by wu-sheng** [comment]
Let's back to this
> I think you miss TraceQueryService? There is a version=1 check there.

**Comment by kezhenxu94** [comment]
> Let's back to this
> 
> > I think you miss TraceQueryService? There is a version=1 check there.

Are you still checking the cached page of GitHub? I’m sure I had already removed it

**Comment by wu-sheng** [comment]
I thought you haven't worked on that. I will recheck today

File: apm-sniffer/apm-agent-core/src/test/java/org/apache/skywalking/apm/agent/core/context/ContextCarrierV2HeaderTest.java [hunk 1]
```diff
@@ -107,37 +58,19 @@ public void testSerializeV2Header() {
...
-                    Assert.fail("unexpected key");
-                }
-            }
-
-        } finally {
-            Config.Agent.ACTIVE_V1_HEADER = false;
+        next = contextCarrier.items();
+        while (next.hasNext()) {
+            next = next.next();
+            //TODO, no BASE64
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/test/java/org/apache/skywalking/apm/agent/core/context/ContextCarrierV2HeaderTest.java]: What is this TODO for?

**Comment by wu-sheng** [comment]
> protocol removal will be done in the protocol repository
> Sorry, I missed this.

@kezhenxu94, what do you mean about `will be`? Are you going to provide another pull request to update the submodule? I prefer it should be done on this one only, to make sure the removal is completed. FYI@JaredTan95

File: oap-server/server-receiver-plugin/skywalking-trace-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/trace/provider/parser/listener/segment/SegmentSpanListener.java [hunk 2]
```diff
@@ -81,10 +82,7 @@ public void parseFirst(SpanDecorator spanDecorator, SegmentCoreInfo segmentCoreI
         segment.setIsError(BooleanUtils.booleanToValue(segmentCoreInfo.isError()));
         segment.setTimeBucket(timeBucket);
         segment.setDataBinary(segmentCoreInfo.getDataBinary());
-        /**
-         * Only consider v1, v2 compatible for now.
-         */
-        segment.setVersion(segmentCoreInfo.isV2() ? 2 : 1);
+        segment.setVersion(segmentCoreInfo.getVersion().number);
```
Comment (wu-sheng) [inline oap-server/server-receiver-plugin/skywalking-trace-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/trace/provider/parser/listener/segment/SegmentSpanListener.java]: `segmentCoreInfo.getVersion().number` Why write in this way? `.number`?

**Pull request merged**